04_numeronym/brvishnu92/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := `[a11y K8s abc]
+`
+	actual := buf.String()
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func TestNumeronyms(t *testing.T) {
+	testCases := map[string]struct {
+		input    []string
+		expected []string
+	}{
+		"test1":  {input: []string{"accessibility", "Kubernetes", "abc"}, expected: []string{"a11y", "K8s", "abc"}},
```

Comment:
It would be better if a different input is used as this input is already tested in TestMainOutput.

Reply:
Modified it :-)